pubspec.yaml

```diff
@@ -1,24 +1,18 @@
 name: bwu_sparkline
-version: 0.0.7
+version: 0.0.7-p1
```

Comment:
It was not intentional, I just wasn't sure how to format the new version, and I was sure you needed to increase the version somehow to push it to pub.  I'll make the change and do another pull request.